src/components/RuntimeManager.js

```diff
@@ -215,7 +218,7 @@ export default class RuntimeManager extends PureComponent {
       notify('warn', 'Please Update Your Cloud Environment', {
         message: h(Fragment, [
           p(['Last year, we introduced important updates to Terra that are not compatible with the older cloud environment associated with this workspace. You are no longer able to save new changes to notebooks using this older cloud environment.']),
-          h(Link, { href: dataSyncingDocUrl, ...Utils.newTabLinkProps }, ['Read here for more details.'])
+          h(Link, { 'aria-label': `Welder cutoff link`, href: dataSyncingDocUrl, ...Utils.newTabLinkProps }, ['Read here for more details.'])
```

Comment:
Since we don't explain what Welder is, I think the "read here for more details" text is probably more helpful.
```suggestion
          h(Link, { href: dataSyncingDocUrl, ...Utils.newTabLinkProps }, ['Read here for more details.'])
```